dataset/src/test/scala/frameless/optimiser/LitTests.scala

```diff
@@ -0,0 +1,172 @@
+package frameless.optimiser
```

Comment:
`optimiser` => `optimizer`

But I want to think about the package names, will try to think about a better package structure / object names later today / this week!

Reply:
sorry, too English, can z it up for sure whilst dropping logging....

Reply:
@chris-twiner No needs to apologize! You've done a tremendous work here 🔥